lib/checks/label/duplicate-img-label.js

```diff
@@ -1,19 +1,17 @@
-const text = axe.commons.text.visibleVirtual(virtualNode, true).toLowerCase();
-if (text === '') {
+if (['none', 'presentation'].includes(node.getAttribute('role'))) {
```

Comment:
Could you use `aria.getRole(node)` instead? I'd like us to step away from moving the role attribute directly.